lib/ansible/modules/find.py

```diff
@@ -154,6 +154,15 @@
             - When doing a C(contains) search, determine the encoding of the files to be searched.
         type: str
         version_added: "2.17"
+    max_matches:
```

Comment:
i would rename `limit` or 'maximum` as this does not relate to a 'match' field, but to all the conditions the other options can set

Reply:
Re-named to `limit`.